mono/utils/mono-error-internals.h

```diff
@@ -5,9 +5,12 @@
 #ifndef __MONO_ERROR_INTERNALS_H__
 #define __MONO_ERROR_INTERNALS_H__
 
+#include <mono/metadata/object-forward.h>
```

Comment:
We do not want to include `metadata/` files from `utils/` since the latter is the code that is not dependent on managed objects.

Reply:
We already do, two lines below.  utils deals with managed exceptions, via raw pointers. I'm just adding handle support right next to the raw pointer support. If this really doesn't belong in utils then a separate change can try to move it. If object-forward.h is too broad, I can make a separate smaller file -- I need handles and exceptions.